src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -2416,25 +2407,39 @@ export class PostgresStorageAdapter implements StorageAdapter {
       ? fieldNames.map((fieldName, index) => `lower($${index + 3}:name) varchar_pattern_ops`)
       : fieldNames.map((fieldName, index) => `$${index + 3}:name`);
     const qs = `CREATE INDEX IF NOT EXISTS $1:name ON $2:name (${constraintPatterns.join()})`;
-    await conn.none(qs, [indexNameOptions.name, className, ...fieldNames]).catch(error => {
-      if (
-        error.code === PostgresDuplicateRelationError &&
-        error.message.includes(indexNameOptions.name)
-      ) {
-        // Index already exists. Ignore error.
-      } else if (
-        error.code === PostgresUniqueIndexViolationError &&
-        error.message.includes(indexNameOptions.name)
-      ) {
-        // Cast the error into the proper parse error
-        throw new Parse.Error(
-          Parse.Error.DUPLICATE_VALUE,
-          'A duplicate value for a field with unique values was provided'
-        );
-      } else {
+    const setIdempodencyFunction = options.setIdempodencyFunction !== undefined ? options.setIdempodencyFunction : false;
```

Comment:
Could you just fix the typo and replace all "idempo`d`ency" with "idempo`t`ency"?

Reply:
done